aten/src/ATen/native/native_functions.yaml

```diff
@@ -288,6 +288,10 @@
 
 - func: conj.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
 
+- func: _conj(Tensor self) -> Tensor
+  use_c10_dispatcher: full
+  variants: function
```

Comment:
I hope this doesn't get traced; that would be `_convolution` all over again.

Reply:
Maybe we want to rename it to `conj_complex()` as it is just a version of `conj()` that **only** accepts complex inputs?

Reply:
@ezyang sorry I am not aware of the issue caused due to `_convolution`. what was it? 
@albanD yeah that makes sense. will do!

Reply:
Alias information for `conj` above is now not correct; you need to give it an alias annotation similar to what `contiguous` has.

Reply:
@anjali411 Basically, `_convolution`, which has a bunch of extra arguments that people typically don't care about, is what is being traced to serialized models, meaning that now it is very difficult to add new parameters to `_convolution` because they are FC-breaking for all models.

conj is considerably less trafficked so the risks of getting it wrong here are lower, but it still seems to me that we should prefer tracing `conj` and not `_conj`. You should test which one we're getting in the trace.

Reply:
Luckily I think the intent here is to keep _conj's signature equivalent to conj, since it's just an autograd hack to only apply autograd to complex inputs to conj. As you point out, @ezyang, _convolution suffers because it's secretly pulling in arguments the user doesn't expect.

Interesting question for what gets traced when ops are composite though: the composite ops or the original symbol. I don't know offhand.

Reply:
Given that the code that handle tracing and autograd was the same until very recently. I expect that only the inner function is traced.